lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1392,6 +1392,11 @@ SPIRVValue *LLVMToSPIRV::transIntrinsicInst(IntrinsicInst *II,
         return transValue(II->getOperand(0), BB);
     }
 
+    // Skip debug info for this call because this call is dropped in SPIR-V.
+    // TODO: remove skipping of debug info when *.annotation call will be
+    //       handled properly during IR <=> SPIR-V translation.
```

Comment:
Since there is no equivalent for this intrinsic in SPIR-V, not sure that there is a proper way to translate it.

```suggestion
    //       handled in a better way during IR <=> SPIR-V translation.
```